webui: provide a notification that NDMP restores are NOT supported by webui

fbergkemper opened the pull request:
webui: provide a notification that NDMP restores are NOT supported by webui
docs: add section that NDMP restores are NOT supported by webui

### Thank you for contributing to the Bareos Project!

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)


### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)

##### General

- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR
- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

pstorz reviewed (changes requested):
Unfortunately webui:admin_restore test fails always.

fbergkemper commented:
> Unfortunately webui:admin_restore test fails always.

There was a misplaced early bsock->disconnect() call, that broke the final restore step. I've fixed and updated the PR. Also I've added another commit for the documentation.

pstorz reviewed (changes requested):
PR Title is misleading. Better: 
webui: provice a notification that NDMP restores are NOT supported by webui.

Also please add a bit more info to the commit descriptions:

docs: add restore section limitations -> docs: add section that NDMP restores are not supported by webui.

webui: provide a restore support notification for NDMP backups -> webui: provide a notification that NDMP restores are not supported by webui. (Same as PR title)

pstorz reviewed (approved):
Looks good. If the build passes with all checks green we can merge.

fbergkemper commented:
> PR Title is misleading. Better: webui: provice a notification that NDMP restores are NOT supported by webui.
> 
> Also please add a bit more info to the commit descriptions:
> 
> docs: add restore section limitations -> docs: add section that NDMP restores are not supported by webui.
> 
> webui: provide a restore support notification for NDMP backups -> webui: provide a notification that NDMP restores are not supported by webui. (Same as PR title)

Done
